fluentd/run.sh

```diff
@@ -123,13 +112,58 @@ if [ "${USE_MUX_CLIENT:-}" = "true" ] ; then
     DIV=`expr $DIV \* 2`
 fi
 
+# Default buffer_type: file if FILE_BUFFER_PATH is specified.
+FILE_BUFFER_PATH=${FILE_BUFFER_PATH:-""}
+if [ "$BUFFER_TYPE" = "file" -a "$FILE_BUFFER_PATH" != "" ] ; then
```

Comment:
also, should it be an error if BUFFER_TYPE = file but no FILE_BUFFER_PATH was given?